version.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
+    <AssemblyVersion>7.0.0.0</AssemblyVersion>
```

Comment:
It's already got an `AssemblyVersion` of 7.0.0.0, is this so we can manage this separately?

Reply:
This is to pin it to `7.0.0.0` for when `7.0.1` happens, which is why I've left it separate, even though it looks duplicated at the moment.

Reply:
Cool, we did use to do this:
https://github.com/justeat/JustSaying/commit/0257f8a68dcc4951fa42708f8aa22a8e3253556c#diff-4b2f402501d23abbede6eac0f47783e4L35

I reverted it cause assembly binding redirects are less of a problem these days, but it likely makes it easier for some consumers still, so worth the effort.

Reply:
I think this is another area that needs guidance on, cause Microsoft have stopped doing this for all of their packages now, even ones that target full framework.

Reply:
It is called out in the guidance on strong-naming:

![image](https://user-images.githubusercontent.com/1439341/47732947-21b03500-dc5f-11e8-8bf6-e71ab424b35e.png)

Reply:
Oh nice, I hadn't seen that